src/classes/ERR_Handler_API.cls

```diff
@@ -68,6 +68,8 @@ global class ERR_Handler_API {
         REL,
         /* @description Recurring Donations context. */
         RD,
+        /* @description Elevate Integration context. */
+        Elevate,
```

Comment:
@psadie-sfdo  just FYI that this is coming in from the Recurring donations side of things